coderedcms/models/page_models.py

```diff
@@ -1147,92 +1137,38 @@ def form_live(self):
         return (self.form_golive_at is None or self.form_golive_at <= timezone.now()) and \
                (self.form_expire_at is None or self.form_expire_at >= timezone.now())
 
-
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        if not hasattr(self, 'landing_page_template'):
-            name, ext = os.path.splitext(self.template)
-            self.landing_page_template = name + '_landing' + ext
-
-    def get_form_fields(self):
-        """
-        Form page expects `form_fields` to be declared.
-        If you want to change backwards relation name,
-        you need to override this method.
-        """
-
-        return self.form_fields.all()
-
-    def get_data_fields(self):
-        """
-        Returns a list of tuples with (field_name, field_label).
-        """
-
-        data_fields = [
-            ('submit_time', _('Submission date')),
-        ]
-        data_fields += [
-            (field.clean_name, field.label)
-            for field in self.get_form_fields()
-        ]
-
-        return data_fields
-
-    def get_form_class(self):
-        fb = self.form_builder(self.get_form_fields())
-        return fb.get_form_class()
-
-    def get_form_parameters(self):
-        return {}
-
-    def get_form(self, *args, **kwargs):
-        form_class = self.get_form_class()
-        form_params = self.get_form_parameters()
-        form_params.update(kwargs)
-
-        return form_class(*args, **form_params)
-
     def get_landing_page_template(self, request, *args, **kwargs):
         return self.landing_page_template
 
-    def get_submission_class(self):
-        """
-        Returns submission class.
-
-        You can override this method to provide custom submission class.
-        Your class must be inherited from AbstractFormSubmission.
-        """
-
-        return FormSubmission
-
-    def process_form_submission(self, request, form):
-        """
-        Accepts form instance with submitted data, user and page.
-        Creates submission instance.
-
-        You can override this method if you want to have custom creation logic.
-        For example, if you want to save reference to a user.
-        """
+    def process_data(self, form, request):
```

Comment:
Please use docstring to document what this function does, and what the arguments are.